File: src/test/java/yomi/com/PersonTest.java
package yomi.com;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class PersonTest {
   @Test//告知
   public void bmiTest() {
     Person person = new Person(66.5f,1.7f);
     Assertions.assertEquals(66.5f/(1.7f*1.7f),person.bmi());

    }

}

File: src/test/kotlin/yomi/com/PersonTest.kt
package yomi.com.kotlin

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test
import yomi.com.Human
import yomi.com.Person


class PersonTest {
    @Test
    fun bmiTest(){
    val human = Human(66.5f,1.7f,"Hank")
        Assertions.assertEquals((66.5f/1.7f*1.7f),human.bmi())
    }
    @Test
    fun personBmiTest(){
        val person = Person(66.5f,1.7f);
        Assertions.assertEquals(66.5f/(1.7f*1.7f),person.bmi())
    }
}
